src/shared/components/formik-fields/ResourceLimitField.tsx

```diff
@@ -31,6 +32,8 @@ const ResourceLimitField: React.FC<ResourceLimitFieldProps> = ({
       <RequestSizeInput
         {...props}
         onChange={(val) => {
+          // run validation manually so that formik uses the latest value
+          validateForm(set(values, props.name, val.value));
           setFieldValue(props.name, val.value);
```

Comment:
Have you tried this workaround? I tried it and it works for me.
```suggestion
         setTimeout(() => setFieldValue(props.name, val.value));
```

Reply:
Thanks! Updated